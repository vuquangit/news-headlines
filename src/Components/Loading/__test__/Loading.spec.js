import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Loading from "..";

describe("Loading", () => {
  it("test default Loading", () => {
    const AppLoading = shallow(<Loading />);
    expect(toJson(AppLoading)).toMatchSnapshot();
  });
});
