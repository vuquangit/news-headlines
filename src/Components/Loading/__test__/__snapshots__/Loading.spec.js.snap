// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading test default Loading 1`] = `
<div
  className="loader"
>
  <div
    className="loader__content"
  >
    Loading...
  </div>
</div>
`;
